packages/ckeditor5-utils/src/emittermixin.ts

```diff
@@ -335,6 +338,64 @@ export default EmitterMixin;
  *
  * @interface Emitter
  */
+export interface Emitter {
```

Comment:
Investigation how to do statically types events is a time-consuming undertaking.

Note for myself: create a ticket to do that and link it here.